tests/runtime/array

```diff
@@ -1,17 +1,17 @@
 NAME array element access - assign to map
-RUN bpftrace -e 'struct MyStruct { int y[4]; } uprobe:./testprogs/array_access:test_struct { $s = (struct MyStruct *) arg0; @x = $s->y[0]; exit(); }'
+RUN bpftrace -v -e 'struct MyStruct { int y[4]; } uprobe:./testprogs/array_access:test_struct { $s = (struct MyStruct *) arg0; @x = $s->y[0]; exit(); }'
```

Comment:
What's the `-v` for here?

Reply:
From the commit message:

> Wait until BPF program attached to call AFTER (we need to run programs with -v for that, otherwise we don't know when the BPF program is attached)

Today when we *start* attaching to probes we output "Attaching to X probes...", but we don't output anything once we finished attaching to those probes, unless we use -v, which outputs "Running..." after attaching to all probes.

Making AFTER run only after the probes are attached was a result of me trying to fix a race condition where some programs would execute faster than we could attach to the proper probes.

I can update the commit message with this extra info if you think that would be helpful.

Reply:
Makes sense, I obviously wasn't reading very carefully yesterday